### PR TITLE
Add OutputDtos new methods

### PR DESCRIPTION
**Pull request by thibault-martinez**
## Change checklist

- [ ] I have followed the contribution guidelines for this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have checked that new and existing unit tests pass locally with my changes
- [ ] I have updated the CHANGELOG.md, if my changes are significant enough

### CODE REVIEW

**Review by samuel-rufi — DISMISSED** [review]

File: bee-block/src/output/mod.rs [hunk 1]
```diff
@@ -300,6 +300,12 @@ pub mod dto {
     };
     use crate::error::dto::DtoError;
 
+    #[derive(Clone)]
```
Comment (Thoralf-M) [inline bee-block/src/output/mod.rs]: What about Debug and Deserialize?

Comment (thibault-martinez) [reply bee-block/src/output/mod.rs]: Yeah can add, doubt Deserialize would be interesting as this is never stored, just temporarily used

Comment (Thoralf-M) [reply bee-block/src/output/mod.rs]: Yeah don't know how you want to use it, if should be provided from bindings we need Deserialize

**Review by pvdrz — APPROVED** [review]

File: bee-block/src/output/mod.rs (hunk 1 above)
Comment (thibault-martinez) [reply bee-block/src/output/mod.rs]: Added them, but shouldn't be an issue anyway

**Review by Thoralf-M — APPROVED** [review]

**Pull request merged**